src/Microsoft.OData.Edm/Schema/EdmContainedEntitySet.cs

```diff
@@ -7,37 +7,52 @@
 namespace Microsoft.OData.Edm
 {
     using System.Collections.Generic;
+    using System.Linq;
 
     /// <summary>
     /// Represents an EDM contained entity set.
     /// </summary>
     internal class EdmContainedEntitySet : EdmEntitySetBase, IEdmContainedEntitySet
     {
+        internal readonly IEdmPathExpression NavigationPath;

```

Comment:
>NavigationPath [](start = 45, length = 14)

nit: should we use a private member named "navigationPath", and expose it thru internal property named "NavigationPath"?